dev.ftb.app.metainfo.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>dev.ftb.app</id>
```

Comment:
`app` isn't a proper app name. As per https://docs.flathub.org/docs/for-app-authors/requirements#application-id